lib/streamlit/elements/layouts.py

```diff
@@ -163,7 +163,7 @@ class name prefixed with ``st-key-``.
 
         if key:
             block_proto.id = compute_and_register_element_id(
-                "container", user_key=key, height=height, border=border
+                "container", user_key=key, form_id=None, height=height, border=border
```

Comment:
is there any generic test we can write to ensure that we do not regress on this? Like some of the generic tests we have in `widgets_test.py` where we iterate over all elements?

Reply:
I removed the parameters for now, added a comment and added a test for container that the ID is computed and contains the key.